extensions/vscode/src/multiStepInputs/newDeployment.ts

```diff
@@ -526,6 +526,16 @@ export async function newDeployment(
     return inputServerUrl(input, state);
   }
 
+  function hasExistingCredential(serverURL: string): boolean {
+    return (
+      credentials.find((credential) => {
+        const existing = normalizeURL(credential.url).toLowerCase();
+        const newURL = normalizeURL(serverURL).toLowerCase();
+        return newURL.includes(existing);
+      }) !== null
+    );
+  }
+
```

Comment:
This would be changed with my request to move this functionality into a common location (with `state.ts`).